### PR TITLE
Makes it impossible for dynamic to roll blood cult

### PR DESCRIPTION
**Pull request by MercuriusGH**
[gamemode]
[based]

It's been more than 2 years since blood cult has gone from a game mode that had a clear start, progression and an end-game to an unfinished dogshit freeform mess on the promise that it will be **fixed later**©
People had a lot of opinions on 3.0 but I'm sure everyone can agree that it was actually finished and functional to a degree
Personally whenever I see blood cult nowadays it makes my stomach turn the same way Time Agent did back when it had barely anything working at all.

It's time to shelve it.
## What this does
Stops dynamic from rolling blood cult

## Why it's good
We'll probably see more of everyone's favourite tried and tested round ending antags with a real end-game firing, like Nuke Ops and Revolution, instead of whatever mess cult is.

## Changelog

:cl:
 * tweak: Makes blood cult impossible to fire naturally

### CODE REVIEW

**Comment by hacker-on-steroids** [comment]
>calling your own PR based

based

**Comment by ghost** [comment]
Suggestion to headmins and host: stake DeityLink's admin flags and collab status on finishing cult and give him ~month to do so. If he pulls himself together and does it then we will have a working gamemode in rotation. If he spergs out like a manchild and ragequits, we won't have him and his unfinished mess of a pet project around anymore.

It's a win-win situation.

**Comment by Pastamadness** [comment]
I'm upvoting this mainly in protest. EDIT: I'm not actually in favor of this PR being merged.
I think it desperately needs a win condition, even if it's an anti-climatic YOU WIN message while ending the round.
Deity could finish fleshing it later but please, end the freeform objective sprawl now. It antagonizes everyone.

**Comment by ErrorsWindows** [comment]
i'm pretty sure i remember deity chimping out a while ago when i put cult in the "gamemodes in development" section on the wiki 🧐 living in the cult four construction site

**Comment by ghost** [comment]
> Deity could finish fleshing it later 

He will absolutely keep postponing it indefinetly unless you force him to and if you are going to enact the Final Solution to the Cultist Question anyway, you might as well make him do it right.

> It's been more than 2 years

**Comment by CorinJace** [comment]
Gone are the days of when cult was a terrible and true danger to a station. Gone are the days when you could summon Nar Nar and everything gets annihilated. Cult now is not even a shadow of what Cult was. Shelve it until Deity gets his act together and finishes the promised changes. Make this his focus.

**Comment by PrimeDSS13** [comment]
> Gone are the days of when cult was a terrible and true danger to a station. Gone are the days when you could summon Nar Nar and everything gets annihilated. Cult now is not even a shadow of what Cult was. Shelve it until Deity gets his act together and finishes the promised changes. Make this his focus.

> > Deity could finish fleshing it later
> 
> He will absolutely keep postponing it indefinetly unless you force him to and if you are going to enact the Final Solution to the Cultist Question anyway, you might as well make him do it right.
> 
> > It's been more than 2 years

tldr: wyci

The current/modern version of cult is in a rather terrible state as a mode (due to being unfinished) and not the most fun so I'd be down for a return to 3.5, but the idea that we could/should "force" someone to work on a mega code project is both entirely outside the realm of reality and also just nonsensical. 

Our codebase is open source with minimal feature contribution, anyone can add to it or remove from it. If you prefer Cult 3, just revert to cult 3. If you like some of the new additions from 4, just add them into the Cult 3 experience. Deity is a player/codemin/contributer, he is and has never been a collab. The thought that you can somehow "force him" to work on a project he's not in the mood to work on (which anyone is free to work on as well) doesn't work because he is 
1. Doing It For Free 
2. Working on multiple other projects for the server such as paint, clothes creation, a ton of random bug fixes and minor additions, etc
3. Not "assigned" to cult or any project (we don't do that at all), he just likes to work on it.

While I hope he continues to work on it and I support making some tweaks/rolling back until then, the suggestion of enacting some sort of "punishment" because he's not working hard enough on a hobby project he does in his free time is insane (not that it ever would occur lol) . All that would do is burn out one of our best contributors.

**Comment by CorinJace** [comment]
> > Gone are the days of when cult was a terrible and true danger to a station. Gone are the days when you could summon Nar Nar and everything gets annihilated. Cult now is not even a shadow of what Cult was. Shelve it until Deity gets his act together and finishes the promised changes. Make this his focus.
> 
> > > Deity could finish fleshing it later
> > 
> > 
> > He will absolutely keep postponing it indefinetly unless you force him to and if you are going to enact the Final Solution to the Cultist Question anyway, you might as well make him do it right.
> > > It's been more than 2 years
> 
> tldr: wyci
> 
> The current/modern version of cult is in a rather terrible state as a mode (due to being unfinished) and not the most fun so I'd be down for a return to 3.5, but the idea that we could/should "force" someone to work on a mega code project is both entirely outside the realm of reality and also just nonsensical.
> 
> Our codebase is open source with minimal feature contribution, anyone can add to it or remove from it. If you prefer Cult 3, just revert to cult 3. If you like some of the new additions from 4, just add them into the Cult 3 experience. Deity is a player/codemin/contributer, he is and has never been a collab. The thought that you can somehow "force him" to work on a project he's not in the mood to work on (which anyone is free to work on as well) doesn't work because he is
> 
>     1. Doing It For Free
> 
>     2. Working on multiple other projects for the server such as paint, clothes creation, a ton of random bug fixes and minor additions, etc
> 
>     3. Not "assigned" to cult or any project (we don't do that at all), he just likes to work on it.
> 
> 
> While I hope he continues to work on it and I support making some tweaks/rolling back until then, the suggestion of enacting some sort of "punishment" because he's not working hard enough on a hobby project he does in his free time is insane (not that it ever would occur lol) . All that would do is burn out one of our best contributors.

Then we should shelve it until someone is willing to complete it. Having a half finished, less than lukewarm gamemode makes the game play unfun. The focus of the threat balance should be on other gamemodes that generate fun and chaos.

**Comment by ghost** [comment]
> the idea that we could/should "force" someone to work on a mega code project is both entirely outside the realm of reality

No it isn't. Make him have a deadline and watch in awe at what a man is capable of once he has something to lose.
>and also just nonsensical. 

No it isn't.
> he is and has never been a collab. 

I stand corrected.
> The thought that you can somehow "force him" to work on a project he's not in the mood to work on (which anyone is free to work on as well) doesn't work because he is

Oh poor him, not in the mood to fix the mess he himself created. If you screw up, it is generally expected from you to clean up your act and correct yourself. Nobody is going to do it for you, for both practical and personal reasons. If he is unwilling to correct his mistakes, then either his work should be discarded entirely until somebody else volunteers to do so (this PR) or he should be made to clean up after himself (my humble suggestion).
>     1. Doing It For Free
>
>     2. Working on multiple other projects for the server such as paint, clothes creation, a ton of random bug fixes and minor additions, etc
> 
>     3. Not "assigned" to cult or any project (we don't do that at all), he just likes to work on it.

Exactly. He clearly has trouble with focusing and self-management, and as such it is only natural to let a strong hand help him out.

> While I hope he continues to work on it and I support making some tweaks/rolling back until then, the suggestion of enacting some sort of "punishment" because he's not working hard enough on a hobby project he does in his free time is insane (not that it ever would occur lol) . All that would do is burn out one of our best contributors.

His hobby is our vidya experience. If he ends up reducing the amount of FUN we receive from playing a videogame, then, like I already said, it is expected from him to raise it back up.

**Comment by gurfan** [comment]
Grow up.

**Comment by Pastamadness** [comment]
Luzha, stop draft dodging and get to Ukraine already.

**Comment by ghost** [comment]
Though I agree my original comment wasn't completely on-topic, it is best to remain silent and not derail the conversation with mudslingling when you run out of worthwhile things to say.

This discussion is over.

**Comment by ErrorsWindows** [comment]
tbh we had changeling removed for less

**Comment by SonixApache** [comment]
cult in its current form is nice variety and has options for fun - antags can do creative things without having to stick to cult things, and the cult stuff is generally accepted to be pretty nice (i know Astral Projection has a lot of fans for one)
though i would much prefer it to have an endgame option, as-is it is good enough to have around

~~as for the discussion in this thread, i'd like to once again ping @d3athrow to ban the ban evader~~
seems i misremembered and this is a different ban evader account, my apologies

**Comment by Kurfursten** [comment]
Thank you for this insightful discussion. I actually do think there were some valuable things gleaned here. It seems to me that what people liked about 3.5 was the satisfying conclusion. Here's what I'm doing to address the problem:

- This PR will remain open for at least a week because I think it serves as a useful place to focus discussion and gather insight into what people liked more about the previous version of cult.
- I have talked to Deity and he is willing to bring back the exciting bloodstone dance summoning ending for Cult which sounds like what people are missing. We will still keep a lot of the solid quality of life improvements from Cult: Narsie's Vacation like mentoring newbies, new rune UI, and objective freedom.

The great thing about this is we have bloodstones, Nar-sie, etc. still in the code. I've talked to Deity about this and he thinks he can have something up in a couple weeks. Then we can have a good place for a public vote: Cult with an end-state or cult as-is.

**Comment by Pastamadness** [comment]
Quick disclaimer that Mercurius blocked Deity and because of that he's incapable of commenting here.

**Comment by PrimeDSS13** [comment]
very cool mercurius
I Will Now Downvote Your PR

**Comment by Blithering** [comment]
_...mgar..._

**Comment by djkramer123** [comment]
>blocking people on github

repoban when

**Comment by ConfusedCactus** [comment]
Lmao he blocked deity so deity can't even comment on this PR to kill cult? Jesus christ. Unironically repoban this guy.

**Comment by Kurfursten** [comment]
This isn't a PR that will go forward under the current mode doctrine given by pomf, which is that if you want to kill a mode you need to code a new one. Blocking deity to prevent him from commenting is pretty stupid and would preclude this PR getting taken seriously if that weren't the case though. I'm not going to start issuing repobans anytime someone blocks another person [unless that's what the headmins want, but it would be a real pain to police], but if you want to revert someone else's content, your PR will not be considered while you have that person blocked. For example, even if you were reverting Hinacode and blocked him, this would apply. The original author must be permitted to state their case if they wish.

However, I still feel that this can be a place for people to give their thoughts on what can be done to improve cult. I already have an action plan stated above and if deity doesn't deliver on cult dance I will port it myself from cult 3.

**Comment by Kurfursten** [comment]
Well I was really hoping there would be more input but I am happy to say that the cult changes are well underway, I have seen the progress updates myself.
